pallets/account-linker/src/lib.rs

```diff
@@ -75,6 +79,14 @@ pub mod pallet {
 	#[pallet::getter(fn btc_addresses)]
 	pub(super) type BitcoinLink<T: Config> =  StorageMap<_, Blake2_128Concat, T::AccountId, Vec<Vec<u8>>, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn polkadot_addresses)]
+	pub(super) type PolkadotLink<T: Config> =  StorageMap<_, Blake2_128Concat, T::AccountId, Vec<T::AccountId>, ValueQuery>;
+
+	#[pallet::storage]
+	#[pallet::getter(fn polkadot_pending)]
+	pub(super) type PolkadotPending<T: Config> =  StorageMap<_, Blake2_128Concat, T::AccountId, (T::AccountId, u32), ValueQuery>;
```

Comment:
If put the link request in the map, then we must deal with the request in order. otherwise the first link request maybe replaced by second one.

Reply:
"intentionally" means you must understand the logic of the implementation, but the user may not.
for example, I want to link two other Polkadot accounts. then I call
link_polkadot  account0, index0
link_polkadot  account1, index1

It will be failed if you use the account0 to accept the link.

Reply:
User has to understand the logic. And the logic is that you have to link it one by one: only after you finish linking and accepting should you do another linking. There will be a user manual for user to follow.